tmk_core/common/eeconfig.h

```diff
@@ -40,6 +40,7 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 // EEHANDS for two handed boards
 #define EECONFIG_HANDEDNESS         			        	(uint8_t *)14
 #define EECONFIG_KEYBOARD                          (uint32_t *)15
+#define EECONFIG_VELOCIKEY                          (uint8_t *)16
```

Comment:
This cannot go here.  It will overlap. 

This NEEDS to be: 
```suggestion
#define EECONFIG_VELOCIKEY                          (uint8_t *)23
```